docs/B-Computations/a-simple-calculation.md

```diff
@@ -1,5 +1,9 @@
 ---
+id: a-simple-calculation
+title: A Simple Calculation
 sidebar_position: 2
+slug: /B-Computations/a-simple-calculation
```

Comment:
Hey @minhhang107 , Nice work on this page!
Something that I noticed was:

Slug should be standardized in small caps and without the "B-"
Check #64 for more details.

Thank you.

Reply:
thanks, I have updated the slug